master/buildbot/test/unit/test_steps_transfer.py

```diff
@@ -563,7 +615,6 @@ def testBasic(self):
         ss.asDict.return_value = dict(revision="12345")
         s.build.getSourceStamp.return_value = ss
 
-        s.step_status = Mock()
```

Comment:
This wasnt really part of this patch, but had noticed that "step_status" was still being referenced here, so I removed it. Tests still pass.